ChunkArray/chunkarray.js

```diff
@@ -0,0 +1,26 @@
+var chunk = function (arr, size) {
+  let start = 0;
+  let ans = [];
+
+  if (size === 0) {
+    return arr;
+  }
+  for (let i = 0; i < Math.ceil(arr.length / size); i++) {
+    
+    start + size > arr.length
+      ? (ans = [...ans, arr.slice(start, arr.length)])
+      : (ans = [...ans, arr.slice(start, size + start)]);
+
+    start = start + size;
```

Comment:
```suggestion
    start += size;
```
use shorthand operator in such cases.